src/main/java/hw_3/Task6.java

```diff
@@ -0,0 +1,19 @@
+package hw_3;
+
+public class Task6 {
+        public static void main(String[] args) {
+            int i = 124;
+            byte b = 66;
+            double d = 77.098;
+
+            Number[] array = new Number[3];
```

Comment:
array type is incorrect.
![Screenshot_13](https://github.com/AntBelyi/ALevel_automation/assets/66726924/fb1e7a42-592b-4312-a310-c2c37ef7d4f0)

Reply:
Done